classes/template/PKPTemplateManager.inc.php

```diff
@@ -1662,16 +1662,25 @@ function getResourcesByContext($resources, $context) {
 	 */
 	function smartyPluckFiles($params, $smarty) {
 
+		// The variable to assign the result to.
+		if (empty($params['assign'])) {
+			error_log('Smarty: {pluck_files} function called without required `assign` param. Called in ' . __FILE__ . ':' . __LINE__);
+			$smarty->assign($params['assign'], array());
```

Comment:
hmm... if the parameter does not exist, what is then the 1. parameter for smarty->assign? does that work?

Reply:
lol, good catch!